### PR TITLE
Fix: node namespace name

### PR DESCRIPTION
**Pull request by pablogs9**

### CODE REVIEW

File: rmw_microxrcedds_c/src/rmw_node.c [hunk 1]
```diff
@@ -91,12 +91,15 @@ rmw_node_t * create_node(
     (uint16_t)domain_id,
     rmw_uxrce_entity_naming_buffer, UXR_REPLACE | UXR_REUSE);
 #else
+  static char xrce_node_name[RMW_UXRCE_NODE_NAME_MAX_LENGTH];
```
Comment (Acuadros95) [inline rmw_microxrcedds_c/src/rmw_node.c:94]: Should we add a 'RMW_UXRCE_NODE_NAME_MAX_LENGTH' check to the namespace + name lenght?

This function just tries to allocate memory without any checks:
https://github.com/micro-ROS/rmw-microxrcedds/blob/a72b8312d9c396b3e3aea38d717a0f33c87a0dd1/rmw_microxrcedds_c/src/rmw_node.c#L59

**Comment by pablogs9** [comment]
Check this before merge: https://github.com/micro-ROS/micro-ROS-Agent/blob/e13375474679c669e454a2846e5063c8a95d0d16/micro_ros_agent/src/agent/graph_manager/graph_manager.cpp#L298

**Comment by pablogs9** [comment]
Related: https://github.com/micro-ROS/micro-ROS-Agent/pull/72

**Review by Acuadros95 — APPROVED** [review]

**Comment by pablogs9** [comment]
@mergify backport foxy galactic

**Pull request merged**